### PR TITLE
update changelog for docker-ce 17.06.1 rc1

### PR DESCRIPTION
**Pull request by andrewhsu**
Built the changelog from the milestone page: https://github.com/docker/docker-ce/milestone/2?closed=1

### CODE REVIEW

**Comment by andrewhsu** [comment]
@thaJeztah @mlaventure

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,32 @@ information on the list of deprecated flags and APIs please have a look at
 https://docs.docker.com/engine/deprecated/ where target removal dates can also
 be found.
 
+## 17.06.1-ce (2017-07-XX)
+
+* (build) Fix add from remote url [#89](https://github.com/docker/docker-ce/pull/89)
```
Comment (thaJeztah) [inline CHANGELOG.md]: Should we put this beneath a `### Builder` header?

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,32 @@ information on the list of deprecated flags and APIs please have a look at
...
+* Prevent a goroutine leak when healthcheck gets stopped [#90](https://github.com/docker/docker-ce/pull/90)
+* Make plugin removes more resilient to failure [#91](https://github.com/docker/docker-ce/pull/91)
```
Comment (thaJeztah) [inline CHANGELOG.md]: Same here; "Plugins" section perhaps

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,32 @@ information on the list of deprecated flags and APIs please have a look at
...
+* Do not error on relabel when relabel not supported [#92](https://github.com/docker/docker-ce/pull/92)
+* Limit max backoff delay to 2 seconds for GRPC connection [#94](https://github.com/docker/docker-ce/pull/94)
+* Fix stderr logging for journald and syslog [#95](https://github.com/docker/docker-ce/pull/95)
+* Use man-pages and docs from docker/cli instead of moby/moby [#96](https://github.com/docker/docker-ce/pull/96)
+* Show progress of replicated tasks before they are assigned [#97](https://github.com/docker/docker-ce/pull/97)
+* Fix log readers can block writes indefinitely [#98](https://github.com/docker/docker-ce/pull/98)
+* (middleware) Redact secret data on "secret create" [#99](https://github.com/docker/docker-ce/pull/99)
+* Fix handling of remote "git@" notation [#100](https://github.com/docker/docker-ce/pull/100)
```
Comment (thaJeztah) [inline CHANGELOG.md]: I think this also could be put under "builder"

File: CHANGELOG.md [hunk 4]
```diff
@@ -5,6 +5,32 @@ information on the list of deprecated flags and APIs please have a look at
...
 be found.
 
+## 17.06.1-ce (2017-07-XX)
+
+* (build) Fix add from remote url [#89](https://github.com/docker/docker-ce/pull/89)
+* Prevent a goroutine leak when healthcheck gets stopped [#90](https://github.com/docker/docker-ce/pull/90)
+* Make plugin removes more resilient to failure [#91](https://github.com/docker/docker-ce/pull/91)
+* Do not error on relabel when relabel not supported [#92](https://github.com/docker/docker-ce/pull/92)
+* Limit max backoff delay to 2 seconds for GRPC connection [#94](https://github.com/docker/docker-ce/pull/94)
+* Fix stderr logging for journald and syslog [#95](https://github.com/docker/docker-ce/pull/95)
```
Comment (thaJeztah) [inline CHANGELOG.md]: Logging

File: CHANGELOG.md [hunk 5]
```diff
@@ -5,6 +5,32 @@ information on the list of deprecated flags and APIs please have a look at
...
+* Use man-pages and docs from docker/cli instead of moby/moby [#96](https://github.com/docker/docker-ce/pull/96)
+* Show progress of replicated tasks before they are assigned [#97](https://github.com/docker/docker-ce/pull/97)
+* Fix log readers can block writes indefinitely [#98](https://github.com/docker/docker-ce/pull/98)
```
Comment (thaJeztah) [inline CHANGELOG.md]: Logging

File: CHANGELOG.md [hunk 6]
```diff
@@ -5,6 +5,32 @@ information on the list of deprecated flags and APIs please have a look at
...
+* (middleware) Redact secret data on "secret create" [#99](https://github.com/docker/docker-ce/pull/99)
+* Fix handling of remote "git@" notation [#100](https://github.com/docker/docker-ce/pull/100)
+* (engine) Vendor forks of containerd + runc + runtime-spec [#102](https://github.com/docker/docker-ce/pull/102)
+* Various completion script updates [#104](https://github.com/docker/docker-ce/pull/104)
+* Various docs fixes [#105](https://github.com/docker/docker-ce/pull/105)
+* Docker wait hangs indefinitely for non-existent container (vendor) [#106](https://github.com/docker/docker-ce/pull/106)
+* Do not add duplicate platform information to service spec [#107](https://github.com/docker/docker-ce/pull/107)
+* Stop trying to load images on an incompatible OS [#108](https://github.com/docker/docker-ce/pull/108)
+* (cli) Don't prune volumes on docker system prune [#109](https://github.com/docker/docker-ce/pull/109)
+* (cli) If `docker swarm ca` is not called with the `--rotate` flag, warn if other flags are passed [#110](https://github.com/docker/docker-ce/pull/110)
```
Comment (thaJeztah) [inline CHANGELOG.md]: `is not called with` -> `is called without` (would that be easier to read?)

File: CHANGELOG.md [hunk 7]
```diff
@@ -5,6 +5,32 @@ information on the list of deprecated flags and APIs please have a look at
...
+* Vndr check engine [#112](https://github.com/docker/docker-ce/pull/112)
```
Comment (thaJeztah) [inline CHANGELOG.md]: "Updated vendored dependencies"

File: CHANGELOG.md [hunk 8]
```diff
@@ -5,6 +5,32 @@ information on the list of deprecated flags and APIs please have a look at
...
+* Show progress of replicated tasks before they are assigned [#97](https://github.com/docker/docker-ce/pull/97)
+* Fix log readers can block writes indefinitely [#98](https://github.com/docker/docker-ce/pull/98)
+* (middleware) Redact secret data on "secret create" [#99](https://github.com/docker/docker-ce/pull/99)
+* Fix handling of remote "git@" notation [#100](https://github.com/docker/docker-ce/pull/100)
+* (engine) Vendor forks of containerd + runc + runtime-spec [#102](https://github.com/docker/docker-ce/pull/102)
+* Various completion script updates [#104](https://github.com/docker/docker-ce/pull/104)
+* Various docs fixes [#105](https://github.com/docker/docker-ce/pull/105)
+* Docker wait hangs indefinitely for non-existent container (vendor) [#106](https://github.com/docker/docker-ce/pull/106)
+* Do not add duplicate platform information to service spec [#107](https://github.com/docker/docker-ce/pull/107)
+* Stop trying to load images on an incompatible OS [#108](https://github.com/docker/docker-ce/pull/108)
```
Comment (thaJeztah) [inline CHANGELOG.md]: would this work? "Improve error message if an image for an incompatible OS is loaded"

File: CHANGELOG.md [hunk 9]
```diff
@@ -5,6 +5,32 @@ information on the list of deprecated flags and APIs please have a look at
...
+* (cli) Don't prune volumes on docker system prune [#109](https://github.com/docker/docker-ce/pull/109)
```
Comment (thaJeztah) [inline CHANGELOG.md]: Thinking of a better description (to also include the `--volumes` flag), not really satisfied, but let me know what you think;

Make pruning volumes optional when running `docker system prune`, and add a `--volumes` flag

File: CHANGELOG.md [hunk 10]
```diff
@@ -5,6 +5,32 @@ information on the list of deprecated flags and APIs please have a look at
...
+* (cli) If `docker swarm ca` is not called with the `--rotate` flag, warn if other flags are passed [#110](https://github.com/docker/docker-ce/pull/110)
+* Vndr check engine [#112](https://github.com/docker/docker-ce/pull/112)
+* Remove OSX cross stuff from main Dockerfile [#113](https://github.com/docker/docker-ce/pull/113)
```
Comment (thaJeztah) [inline CHANGELOG.md]: Remove unused Mac OS X cross compilation code

File: CHANGELOG.md [hunk 11]
```diff
@@ -5,6 +5,32 @@ information on the list of deprecated flags and APIs please have a look at
...
+* (swarmkit) Cluster update and memory issue fixes [#114](https://github.com/docker/docker-ce/pull/114)
+* Set ping version even on error [#115](https://github.com/docker/docker-ce/pull/115)
```
Comment (thaJeztah) [inline CHANGELOG.md]: Fix API version negotiation not working if the daemon returns an error

File: CHANGELOG.md [hunk 12]
```diff
@@ -5,6 +5,32 @@ information on the list of deprecated flags and APIs please have a look at
...
+* Fix stderr logging for journald and syslog [#95](https://github.com/docker/docker-ce/pull/95)
+* Use man-pages and docs from docker/cli instead of moby/moby [#96](https://github.com/docker/docker-ce/pull/96)
+* Show progress of replicated tasks before they are assigned [#97](https://github.com/docker/docker-ce/pull/97)
+* Fix log readers can block writes indefinitely [#98](https://github.com/docker/docker-ce/pull/98)
+* (middleware) Redact secret data on "secret create" [#99](https://github.com/docker/docker-ce/pull/99)
+* Fix handling of remote "git@" notation [#100](https://github.com/docker/docker-ce/pull/100)
+* (engine) Vendor forks of containerd + runc + runtime-spec [#102](https://github.com/docker/docker-ce/pull/102)
+* Various completion script updates [#104](https://github.com/docker/docker-ce/pull/104)
+* Various docs fixes [#105](https://github.com/docker/docker-ce/pull/105)
+* Docker wait hangs indefinitely for non-existent container (vendor) [#106](https://github.com/docker/docker-ce/pull/106)
```
Comment (thaJeztah) [inline CHANGELOG.md]: Fix `docker wait` hanging if the container does not exist

File: CHANGELOG.md [hunk 13]
```diff
@@ -5,6 +5,32 @@ information on the list of deprecated flags and APIs please have a look at
...
+* Do not error on relabel when relabel not supported [#92](https://github.com/docker/docker-ce/pull/92)
+* Limit max backoff delay to 2 seconds for GRPC connection [#94](https://github.com/docker/docker-ce/pull/94)
+* Fix stderr logging for journald and syslog [#95](https://github.com/docker/docker-ce/pull/95)
+* Use man-pages and docs from docker/cli instead of moby/moby [#96](https://github.com/docker/docker-ce/pull/96)
+* Show progress of replicated tasks before they are assigned [#97](https://github.com/docker/docker-ce/pull/97)
+* Fix log readers can block writes indefinitely [#98](https://github.com/docker/docker-ce/pull/98)
+* (middleware) Redact secret data on "secret create" [#99](https://github.com/docker/docker-ce/pull/99)
+* Fix handling of remote "git@" notation [#100](https://github.com/docker/docker-ce/pull/100)
+* (engine) Vendor forks of containerd + runc + runtime-spec [#102](https://github.com/docker/docker-ce/pull/102)
+* Various completion script updates [#104](https://github.com/docker/docker-ce/pull/104)
```
Comment (thaJeztah) [inline CHANGELOG.md]: Update Bash completion scripts

File: CHANGELOG.md (hunk 1 above)
Comment (thaJeztah) [inline CHANGELOG.md]: Fix a regression, where `ADD` from remote URL's extracted archives

**Pull request merged**